src/config.rs

```diff
@@ -551,6 +552,11 @@ pub(crate) struct BackportRuleConfig {
     pub(crate) add_labels: Vec<String>,
 }
 
+#[derive(Default, PartialEq, Eq, Debug, serde::Deserialize)]
```

Comment:
Could you please add some doc comment on top of this? It would be nice to have some simple programmatic documentation about the individual config options in this file.